src/legacy/router.js

```diff
@@ -109,79 +103,6 @@ function getCommonMetadata(ctx) {
   };
 }
 
-async function getCdkV2Result(destName, event, feature) {
```

Comment:
@koladilip Are we completely removing the support of CDKv2 vs native way of testing ? In complex & majorly used destinations, we should be still needing this from the data testing perspective. Any other alternative you had ?

Reply:
@koladilip will come with plan of testing CDKv2 migration